extensions/pom.xml

```diff
@@ -30,7 +30,6 @@
         <module>jsonb</module>
 
         <!-- HTTP -->
-        <module>vertx-core</module>
```

Comment:
I would have moved the vertx module here.

Reply:
Hm, it's under the `<!-- Reactive -->` group which is probably more appropriate? The `vertx-web` module (which I plan to remove in the second part of PR) could be moved under the `<!-- HTTP -->` group though.